### ISSUE TITLE
reverse props

### ISSUE THREAD

**Issue by MalikWhitten67** [MalikWhitten67/html-dox#7]
- Implement reverse props
<blockquote>
Example
</blockquote

 ```html
<!-- Nav.html -->
<nav props="url:text">
    <ul>
        <li>
            <a href="/{{url}}">{{text}}</a>
        </li>
    </ul>
</nav>

<!-- main.html -->
<import src="./components/nav.html" exports="nav"></import>
<main>
   <!-- text is initially set to "hello world" and url is set to "this/that" -->
    <nav text="hello world" url="this/that"></nav>
    <h1>Hello world!</h1>
    <h2>Welcome to <span class="orange">HTML DOX</span></h2>
</main>

<export>
    main
</export>



```
```js
dox.querySelector('nav').setProps('text', 'hi')
```

**Comment by MalikWhitten67** [comment]
Implemented as of [v1.1.4](https://github.com/MalikWhitten67/html-dox/releases/tag/v1.1.4)

**Comment by MalikWhitten67** [comment]
Implemented as of [v1.1.4](https://github.com/MalikWhitten67/html-dox/releases/tag/v1.1.4)
